dae-alright/src/components/Navigation/NavLoggedIn.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "../Navigation/NavLoggedIn.scss";
+import {Button} from 'react-bootstrap';
+
+
+class NavLoggedIn extends React.Component{
```

Comment:
Make this a functional component please, moreover, I don't think it's needed because there's a custom button you can import in the `Navigation` component for this

Reply:
Done